### PR TITLE
Fix additional load time warnings

### PR DESCRIPTION
**Pull request by davidkline-ms**
There were some additional scenes that had multiple EventSystem components on the MainCamera and one lingering missing ConstraintManager.

This change fixes all known instances of both of these issues.

### CODE REVIEW

File: Assets/MRTK/Examples/Demos/Solvers/Scenes/FollowSolverExample.unity [hunk 1]
```diff
@@ -1654,21 +1637,6 @@ Camera:
...
---- !u!114 &1087739262
-MonoBehaviour:
-  m_ObjectHideFlags: 0
-  m_CorrespondingSourceObject: {fileID: 0}
-  m_PrefabInstance: {fileID: 0}
-  m_PrefabAsset: {fileID: 0}
-  m_GameObject: {fileID: 1087739255}
-  m_Enabled: 1
-  m_EditorHideFlags: 0
-  m_Script: {fileID: -619905303, guid: f70555f144d8491a825f0804e09c671c, type: 3}
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Demos/Solvers/Scenes/FollowSolverExample.unity:1666]: Same question here one the 2 vs 3 existing event systems

**Review by keveleigh — COMMENTED** [review]
I wrote the question twice, but then realized several more scenes looked the same so I'll just mention that here haha

File: Assets/MRTK/Examples/Demos/PulseShader/Scenes/PulseShaderSpatialMeshExample.unity [hunk 2]
```diff
@@ -679,21 +662,6 @@ Camera:
...
---- !u!114 &135207088
-MonoBehaviour:
-  m_ObjectHideFlags: 0
-  m_CorrespondingSourceObject: {fileID: 0}
-  m_PrefabInstance: {fileID: 0}
-  m_PrefabAsset: {fileID: 0}
-  m_GameObject: {fileID: 135207081}
-  m_Enabled: 1
-  m_EditorHideFlags: 0
-  m_Script: {fileID: -619905303, guid: f70555f144d8491a825f0804e09c671c, type: 3}
```
[the remark this answers is not in the document]
Comment (davidkline-ms) [reply Assets/MRTK/Examples/Demos/PulseShader/Scenes/PulseShaderSpatialMeshExample.unity:691]: i dont know what happened.... i updated the file and it should not have one (i believe)

File: Assets/MRTK/Examples/Demos/Solvers/Scenes/FollowSolverExample.unity (hunk 1 above)
Comment (davidkline-ms) [reply Assets/MRTK/Examples/Demos/Solvers/Scenes/FollowSolverExample.unity:1666]: yeah

**Review by keveleigh — APPROVED** [review]

**Pull request merged**